packages/components/Button/src/ButtonTokens.ts

```diff
@@ -12,6 +13,7 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
     medium: {
       padding: Platform.select({
         android: globalTokens.spacing.xs,
+        ios: globalTokens.spacing.xs,
         default: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
       }),
```

Comment:
Revisiting: Can we move this to platform specific files? Or if those don't exist and that causes issues, at least leave a comment why w need this at the moment?

Reply:
Yes I was planning to fix this, this was just when I merged in their changes